src/test/java/org/opensearch/neuralsearch/processor/rerank/MLOpenSearchRerankProcessorTests.java

```diff
@@ -223,6 +230,49 @@ public void testRerankContext_whenQueryTextPathIsBadPointer_thenFail() throws IO
             .equals(QueryContextSourceFetcher.QUERY_TEXT_PATH_FIELD + " must point to a string field"));
     }
 
+    public void testRerankContext_whenQueryTextPathIsExceeedinglyManyCharacters_thenFail() throws IOException {
```

Comment:
Could you use `@SneakyThrows` for test case? We haven't documented publicly but we want to use `@SneakyThrows` in test case and `throws *Exception` in others.
```suggestion
    @SneakyThrows
    public void testRerankContext_whenQueryTextPathIsExceeedinglyManyCharacters_thenFail() {
```